src/Routes/RosSystemDetail/ros-details-page.scss

```
@@ -12,4 +12,7 @@
       }
     }
   }
+}
```

@r14chandra here is the styles you can use with the `DescriptionList` 

```suggestion
.ros-system-info {
  .pf-c-description-list.expanded-row {    
    dt {
      font-weight: var(--pf-global--FontWeight--normal);
      // width: 120px;
      &:after {
        content: ":";
      }
    }
  }

  .pf-c-description-list__group {
    row-gap: 0;
  }
}
```